docs/data/material/varnish/colors/ColorValues.js

```diff
@@ -6,7 +6,8 @@ import * as React from 'react';
 import PropTypes from 'prop-types';
 import { useTheme, styled } from '@mui/material/styles';
 import Typography from '@mui/material/Typography';
-import { CopyToClipboardButton } from '@allenai/varnish';
+import { CopyToClipboardButton } from '@allenai/varnish2';
```

Comment:
i had to remove icon from copytoclipboard, i would like to put it back some day if we figure out how to get the build to properly resolve the icons